sdks/java/extensions/protobuf/src/main/java/org/apache/beam/sdk/extensions/protobuf/ProtoByteBuddyUtils.java

```diff
@@ -538,7 +538,7 @@ static <ProtoT> FieldValueGetter<ProtoT, OneOfType.Value> createOneOfGetter(
       return builder
           .visit(new AsmVisitorWrapper.ForDeclaredMethods().writerFlags(ClassWriter.COMPUTE_FRAMES))
           .make()
-          .load(ReflectHelpers.findClassLoader(), ClassLoadingStrategy.Default.INJECTION)
+          .load(ReflectHelpers.findClassLoader(), getClassLoadingStrategy(protoClass))
```

Comment:
while the change of all other files follow the same pattern, the use case here looks slightly different. Would you mind providing some hint to the changes in ProroByteBuddyUtil ?

Or if there is anything not sure, we can leave this file untouched. I just recall the last time of this effort  (#23210) broke some test and had some back and forth. I was wondering if any remaining usage Default.INJECTION not removed in #25578 was due to that it is nontrivial

Reply:
the class used was the one used for package injection in the helper method.
But since I'm not sure how to best test this and it wasn't directly causing issues for me I'm removing from this PR.